src/components/accordionItem/AccordionItem.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Dropdown } from '../index';
+import * as S from './AccordionItem.styles';
+
+const AccordionItem = ({ data, toggle, clicked }) => {
+  const { id, title, about } = data;
+  const isClicked = clicked === id;
+
+  return (
+    <>
+      <S.Wrap onClick={() => toggle(id)}>
+        <S.Element>{id}</S.Element>
+        <h2>{title}</h2>
+        <S.BorderLine active={isClicked} />
+      </S.Wrap>
+      {isClicked && <Dropdown data={about} />}
```

Comment:
just a note - i'm a bit confused with name "dropdown". it means some content inside accordion here, but "dropdown" usually means some area with clickable options list

Reply:
I had in mind that is like drop down menu, but I can change to `Content` or `DropdownContent`. Which one would be better and cleaner?

Reply:
Changed to `AccordionItemContent` :)